src/executables/util_cs.c

```diff
@@ -1898,6 +1904,10 @@ killtran (UTIL_FUNCTION_ARG * arg)
   /* disable password, if don't use kill option */
   if (isbatch == 0)
     {
+      if (dba_password != NULL)
+	{
+	  goto print_killtran_usage;
+	}
       dba_password = NULL;
```

Comment:
This statement is not required. The value of **dba_password** is already NULL.
If it was not NULL, it would have jumped to the **print_killtran_usage** label.

Reply:
OK, I will remove this code.